src/lib/address_resolve/AddressResolve_DefaultImpl.cpp

```diff
@@ -35,6 +38,8 @@ void NodeLookupHandle::ResetForLookup(System::Clock::Timestamp now, const NodeLo
 
 void NodeLookupHandle::LookupResult(const ResolveResult & result)
 {
+    MATTER_LOG_NODE_DISCOVERED(chip::Tracing::DiscoveryInfoType::kIntermediateResult, &GetRequest().GetPeerId(), &result);
```

Comment:
Why the `chip::` prefixes here and below? We're inside `namespace chip`.

Reply:
copy and paste. Fixed.